src/semantic_release/commit_parser/conventional/parser.py

```diff
@@ -215,17 +164,20 @@ def __init__(self, options: ConventionalCommitParserOptions | None = None) -> No
                             commit_type_pattern.pattern + r"\b",  # prior to commit type
                         ],
                     ),
-                    flags=re.MULTILINE,
+                    flags=MULTILINE,
                 ),
                 # move commit type to the start of the line
                 r"\1",
             ),
         }
 
-    @staticmethod
-    def get_default_options() -> ConventionalCommitParserOptions:
+    def get_default_options(self) -> ConventionalCommitParserOptions:
         return ConventionalCommitParserOptions()
 
+    def log_parse_error(self, commit: Commit, error: str) -> ParseError:
```

Comment:
Note that all of the instances of this need to be replaced with the following, otherwise it will fail at runtime (since this is just a type hint import):
```suggestion
    def log_parse_error(self, commit: "Commit", error: str) -> ParseError:
```

Reply:
This needs to happen in numerous places